src/components/patterns/pill-card.tsx

```diff
@@ -66,9 +66,11 @@ const PillCard = ({imageUrl, imageAlt, videoUrl, isArticle, children, bgColor, .
 
       <div
         className={twMerge(
-          "rs-px-3 flex flex-col pb-[125px] pt-20 @7xl:rs-px-4 @2xl:pb-[175px] @3xl:pb-[225px] @4xl:pb-[300px]",
+          "rs-px-3 flex flex-col pb-[125px] pt-20 @7xl:rs-px-4 @2xl:pb-[175px] @3xl:pb-[225px] @4xl:pb-[300px] [&_p>a]:text-black-true hocus:[&_p>a]:text-[#00365C]",
```

Comment:
Slipping in the accessibility color change as well 🤔 Does this darker digital blue color change work for the hover/focus state?

https://webaim.org/resources/contrastchecker/?fcolor=00365C&bcolor=F9A44A
https://webaim.org/resources/contrastchecker/?fcolor=00365C&bcolor=A6B168
https://webaim.org/resources/contrastchecker/?fcolor=00365C&bcolor=F4795B

Reply:
Looks like it does, but the links on the "Spirited Light" (2.46) and the "Olive Light" (2.86) still don't work